index.js

```diff
@@ -44,49 +34,126 @@ app.get('/docs', (req, res) => {
   res.send(apiDocs);
 });
 
-app.get('/status', (req, res) => {
-  res.send({
-    'statusCode': wallet.state,
-    'statusMessage': humanState[wallet.state],
-    'network': wallet.network,
-    'serverUrl': wallet.server,
-    'serverInfo': wallet.serverInfo,
+app.post('/start', (req, res) => {
+  // We expect the user to send the seed he wants to use
+  // if not sent, we use the 'default' key
+  const seedKey = req.body.seedKey || 'default';
```

Comment:
I think we should return an error if the seedKey is not explicitly given.

Reply:
Makes sense, changed it.